spec/requests/objects_controller_content_diff_spec.rb

```diff
@@ -61,7 +61,8 @@
 
       context 'when ArgumentError from MoabStorageService' do
         it 'returns 400 response code with details' do
-          post content_diff_object_url(id: prefixed_druid), params: { content_metadata: content_md, subset: 'unrecognized' }
+          params = { content_metadata: content_md, subset: 'unrecognized' }
```

Comment:
this also is not like the others (one line) and I don't care

Reply:
iirc, rubocop was complaining about line length, so this was meant to break up the line to appease rubocop.